.github/workflows/docbuild.yml

```diff
@@ -1,6 +1,6 @@
 name: docs
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
I think, it's better to keep both, for example in a scenario where a developer works on a pull requests's review comments, he makes the changes and push the changes for further review, at those times too the workflows should be executed!!

Reply:
Not sure why, as we have protected main branch and all pushes go via a pull-request. Why do things double, and fill the test-field with double entries?

Reply:
I don't understand what the purpose of doing the same test twice is. If it fails once, it will fail the second time; if it passes once, it will pass the second time. It is additional and has no additional benefit

Reply:
Let's take a scenario:

- a developer creates a pull request and during the creation of the PR all the tests succeeded and the review process starts. During the review process, developer address some review comments and push more changes which will break some tests, so during this time when he push his breaking changes, will the pytest workflow be executed and failed? (especially after when you merge these changes made to the workflows)